tests/test_air_absorption.py

```diff
@@ -0,0 +1,45 @@
+DEBUG = False
+
+from audiomentations import AirAbsorption
+import numpy as np
+import pytest
+import scipy
+import scipy.signal
+
+
+def get_chirp_test(sample_rate, duration):
+    """Create a `duration` seconds chirp from 0Hz to `nyquist frequency`"""
+    n = np.arange(0, duration, 1 / sample_rate)
+    samples = scipy.signal.chirp(n, 0, duration, sample_rate // 2, method="linear")
+    return samples.astype(np.float32)
+
+
+class TestWindAbsorptionTransform:
```

Comment:
You could maybe also test that you chose the right key in `air_absorption_table` ?

Reply:
Can you give an example of what you mean?

Reply:
I mean that you map the input to the relevant key

Reply:
The key is not exposed though and I would prefer not to.